src/SFA.DAS.EmployerFeedback.Application/Commands/SubmitEmployerFeedback/SubmitEmployerFeedbackCommand.cs

```diff
@@ -0,0 +1,16 @@
+﻿using MediatR;
+using SFA.DAS.EmployerFeedback.Domain.Types;
+using SFA.DAS.EmployerFeedback.Infrastructure.Api.Types;
+
+namespace SFA.DAS.EmployerFeedback.Application.Commands.SubmitEmployerRequest
```

Comment:
Namespace not matching with folder structure

Reply:
Fixed